vsphere/internal/helper/network/network_helper.go

```diff
@@ -197,3 +198,43 @@ func dvsFromUUID(client *govmomi.Client, uuid string) (*object.VmwareDistributed
 
 	return dvsFromMOID(client, resp.Returnval.Reference().Value)
 }
+
+func FromName(client *vim25.Client, name string, dc *object.Datacenter, additionalFilters map[string]string) (object.NetworkReference, error) {
```

Comment:
This is pretty generalized and an exported function name like some of the others in the network package. Suggest adding a description comment above the function.